app/controllers/register/template.hbs

```diff
@@ -0,0 +1,30 @@
+<div class="materialize-me">
+  <div class="container register">
+    <div class="center-align register-logo"></div>
+    <div class="row">
+      <form class="col push-4 s12">
+        <div class="row">
+          <div class="input-field col s12 center-align">
+            <Input @value={{this.passwordText}} type={{if this.showPassword "text" "password"}} @input={{action "valChange"}} class="validate"
+              id="password" />
+            <label for="password">Password</label>
+            <label onclick={{action "showPasswordClick"}} class="right label-icon show-password" for="password"><i class="fas {{if this.showPassword "fa-eye-slash" "fa-eye"}}"></i></label>
```

Comment:
Use the `on` modifier – https://blog.emberjs.com/2019/03/06/coming-soon-in-ember-octane-part-4.html